### PR TITLE
july preview

### PR DESCRIPTION
**Pull request by xiangyan99**
Turn back to preview bits for July

### CODE REVIEW

**Comment by xiangyan99** [comment]
/adp run python - search - tests

File: sdk/search/azure-search-documents/samples/async_samples/sample_semantic_search_async.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    endpoint = os.getenv("AZURE_SEARCH_SERVICE_ENDPOINT")
+    index_name = os.getenv("AZURE_SEARCH_INDEX_NAME")
+    api_key = os.getenv("AZURE_SEARCH_API_KEY")
+
+    credential = AzureKeyCredential(api_key)
+    client = SearchClient(endpoint=endpoint,
+                          index_name=index_name,
+                          credential=credential)
+    results = list(client.search(search_text="luxucy", query_language="en-us", speller="lexicon"))
```
Comment (kristapratico) [inline sdk/search/azure-search-documents/samples/async_samples/sample_semantic_search_async.py:38]: I don't think list() will work with async

File: sdk/search/azure-search-documents/samples/async_samples/sample_semantic_search_async.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    for result in results:
+        print("{}\n{}\n)".format(result["HotelId"], result["HotelName"]))
+    # [END speller_async]
+
+async def semantic_ranking():
+    # [START semantic_ranking_async]
```
Comment (kristapratico) [inline sdk/search/azure-search-documents/samples/async_samples/sample_semantic_search_async.py:45]: do these sphinx declarations need to be added to the client docstring?

**Review by kristapratico — APPROVED** [review]

File: sdk/search/azure-search-documents/samples/async_samples/sample_semantic_search_async.py (hunk 2 above)
Comment (xiangyan99) [reply sdk/search/azure-search-documents/samples/async_samples/sample_semantic_search_async.py:45]: This one we have not decided yet. :)

**Pull request merged**